### DISCUSSION TITLE
关于快照审查的一个疑问

### DISCUSSION THREAD

**Discussion by AIsouler** [gkd-kit/gkd#593] [General]
在快照https://i.gkd.li/i/13635579中，
选择器```View[childCount=2] > [text="广告"] + @View[clickable=true]```能够选中两个节点，
选择器```View[childCount=2] > [text="广告"] + @View[clickable=true] <<n [id="com.gamersky:id/contentWebView"]```只能选中一个节点，
选择器```View[childCount=5] >4 View[childCount=2] > [text="广告"] + @View[clickable=true] <<n [id="com.gamersky:id/contentWebView"]```能选中另外一个节点，
请问这是bug吗？

**Comment by lisonge** [comment]
你可以参考 <https://github.com/orgs/gkd-kit/discussions/569#discussioncomment-8885719>
